src/main/java/yoda/task/TaskList.java

```diff
@@ -25,39 +25,39 @@ public TaskList(List<Task> taskList) {
      * @return Task requested by user
      */
     public Task accessTask(int taskNumber) {
-        return taskList.get(taskNumber);
+        return tasks.get(taskNumber);
     }
 
     /**
      * Adds a task to the list.
      * @param task Task to be added.
      */
     public void addTask(Task task) {
-        taskList.add(task);
+        tasks.add(task);
     }
 
     /**
      * Marks a task in the list as done.
      * @param taskNumber Position of task in the list.
      */
     public void markTaskAsDone(int taskNumber) {
-        taskList.get(taskNumber).markAsDone();
+        tasks.get(taskNumber).markAsDone();
     }
 
     /**
      * Deletes a task from the list.
      * @param taskNumber Position of task in list.
      */
     public void deleteTask(int taskNumber) {
-        taskList.remove(taskNumber);
+        tasks.remove(taskNumber);
     }
 
     /**
      * Gets the number of tasks in the list.
      * @return The number of tasks in the list
      */
     public int length() {
```

Comment:
As a getter used to fetch the numerical size of the task list, there are a few pointers to note:

1. Include the word "get" in the method name
2. Be more specific in what length are you exactly referring to

Hence, my suggestion would be "getTaskLength" or "getTaskSize".